source/ncengine/graphics2/NcGraphicsImpl2.cpp

```diff
@@ -74,16 +74,13 @@ namespace nc::graphics
 
         if (graphicsSettings.enabled)
         {
-            auto ncAsset = modules.Get<asset::NcAsset>();
             auto ncWindow = modules.Get<window::NcWindow>();
-            NC_ASSERT(ncAsset, "NcGraphics requires NcAsset to be registered before it.");
```

Comment:
This assert should stay. We can discard the value like with NcScene, if its currently unused.